src/components/DeleteButton.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { View, Pressable, StyleSheet, type StyleProp, type ViewStyle } from 'react-native'
+import { KeyboardContext } from '../contexts/KeyboardContext'
+import { Icon } from './Icon'
+
+type DeleteButtonStyles = {
+  containerStyle?: StyleProp<ViewStyle>
+  buttonStyle?: StyleProp<ViewStyle> | ((state: { pressed: boolean }) => StyleProp<ViewStyle>)
+  iconNormalColor?: string
+  iconActiveColor?: string
+}
+
+type CustomButtonType = {
+  customButtonPressHandler?: () => void
+  style?: DeleteButtonStyles
+}
```

Comment:
Since the button is a Pressable, what about just extending the PressableProps type, sth like this?

```suggestion
type CustomButtonType = {
  containerStyle?: StyleProp<ViewStyle>
  iconNormalColor?: string
  iconActiveColor?: string
} & PressableProps
```

If one decides to use this component, such implementation allows for passing other Pressable-specific props or simply just the testID etc? wdyt? 🤔

Reply:
Good point. I changed it like you said